test/Npgsql.Tests/ReaderNewSchemaTests.cs

```diff
@@ -13,43 +14,37 @@ namespace Npgsql.Tests
     /// Note that this API is also available on .NET Framework.
     /// For the old DataTable-based API, see <see cref="ReaderOldSchemaTests"/>.
     /// </summary>
+    [TestFixture(false)] // Sync
```

Comment:
Rather than a boolean flag (which requires comments to explain the meaning), we can add an AsyncOrNot enum (see the existing PreparedOrNot/PooledOrNot ones). This also shows up nicely in test runners.